Assets/HoloToolkit-Examples/SpatialUnderstanding/SpatialUnderstanding-FeatureOverview/ProjectSettings/UnityAdsSettings.asset

```diff
@@ -0,0 +1,11 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!292 &1
+UnityAdsSettings:
```

Comment:
I set this up to be a stand-alone example project. So, "SpatialUnderstanding-FeatureOverview" is the Unity project path for the example. It looks like people are sometimes just adding misc files to be dropped into a project (spatialMapping & ColorPicker) & sometimes a full assets folder (GazeRuler). 

There's nothing too special about my project settings - just the things that are required for holotoolkit. If we don't want a full project, I could drop it and just do the asset folder (like GazeRuler).